docs/src/content/navigation.yaml

```diff
@@ -1,7 +1,11 @@
 navGroups:
   - groupName: Getting started
     items:
-      - label: Automated setup
+      - label: Introduction
+        link:
+          discriminant: page
+          value: introduction
+      - label: Automated Setup
```

Comment:
Should we use sentence or title here? Seems to be a bit of a mix (topical topic for today)

Reply:
Hmm yes.. I'll prob give Tuan a shout re: this and see

Reply:
Fixed, FYI we're following https://balance.reckon.com/content/capitalisation